grails-forge-core/src/main/java/org/grails/forge/feature/test/GebWithWebDriverBinaries.java

```diff
@@ -70,12 +70,7 @@ public boolean supports(ApplicationType applicationType) {
 
     @Override
     public String getDocumentation() {
-        return "https://github.com/grails3-plugins/geb#readme";
-    }
-
-    @Override
-    public String getThirdPartyDocumentation() {
-        return "https://www.gebish.org/manual/current/";
+        return "https://github.com/apache/groovy-geb";
```

Comment:
I think https://www.gebish.org/manual/current is the right link for third-party documentation.

Reply:
@matrei I was suggesting to define links only against Github and delegate to the README.md of each project to link to third party documentation. This would be more future proof since each project can manage its own documentation links. Let me know if that makes sense to you.

Reply:
@gsartori I think that can be a good idea for many projects, like grails plugins. But Geb is famous for it's outstanding documentation. I think it deserves a direct link.